wetterdienst/service.py

```diff
@@ -145,17 +148,18 @@ def dwd_values(
     :param tidy:        Whether to return data in tidy format. Default: True.
     :return:
     """
-    if product not in ["observation", "mosmix"]:
-        return HTTPException(status_code=404, detail=f"product {product} not found")
+    provider = "dwd"
+    if kind not in ["observation", "mosmix"]:
+        return HTTPException(status_code=404, detail=f"product {kind} not found")
 
-    if station is None:
+    if stations is None:
         raise HTTPException(
             status_code=400, detail="Query argument 'station' is required"
```

Comment:
Maybe also adjust the error message here?

```suggestion
            status_code=400, detail="Query argument 'stations' is required"
```